request.go

```diff
@@ -134,16 +134,6 @@ func (r *Requester) SetClient(client *http.Client) *Requester {
 	return r
 }
 
-func (r *Requester) parseQueryString(queries map[string]string) string {
```

Comment:
In looking at the [`README`](https://github.com/remyoudompheng/go-misc/tree/master/deadcode), it says the following:

# Limitations
* Self-referential unused code is not currently reported
* A single package can be tested at a time
* Unused methods are not reported